src/yaml/wmp/mega-textures.yaml

```diff
@@ -0,0 +1,22 @@
+group: wmp
+name: mega-textures
+version: "1.0.2"
+subfolder: 100-props-textures
+info:
+  summary: Mega Textures
+  description: |-
+    This new Mega Texture set contains the textures from all the previosly released WMP Texture packs (**WMP Textures 1-10** and the **WMP Mega Textures**) and it also has the textures from the previous **wmp\_spam\_freight\_props2**, **rfy\_warehouse** and **rfy\_cardealer** props, which contained some textures.
+  author: DocRorlach, WMP Team, pLynn, RFY Team
+  website: https://community.simtropolis.com/files/file/33626-wmp-mega-textures-v102/
+  images:
+    - https://www.simtropolis.com/objects/screens/monthly_2020_05/5ecbe56fdd4c5_WMPMegaTextures.jpg.d397bb120692a2a37f1b950184196fc0.jpg
+dependencies:
+  - wmp:essentials
+assets:
+  - assetId: stex-custodian-wmp-mega-textures-v102
+
+---
+assetId: stex-custodian-wmp-mega-textures-v102
+version: "1.0.2"
```

Comment:
The `v102` version string shouldn't usually be part of the `assetId`, as it's already tracked by the `version` attribute. If the STEX file is updated, it's title is going to change, but the `assetId` should ideally be persistent. It's not a big issue practically though.

Reply:
That's because I parse the asset id directly from the url, but I'll probably adapt to filter out version strings automatically. I also notice that the asset id still includes stex-custodian. I'll remove that as well.